[DFMD-8] Research Programming Languages

MoyinoluwaA opened the pull request.

Ireti203 reviewed (commented):
You deleted some files in another scholar's folder. Is there a reason you are doing this?

MoyinoluwaA commented:
Good morning. There was a folder it said I added stuffs and I didn't. So I
saw your comment on the slack channel. That's why I deleted it.

On Wed, 10 Feb 2021, 07:56 Ireti203, <notifications@github.com> wrote:

> *@Ireti203* commented on this pull request.
>
> You deleted some files in another scholar's folder. Is there a reason you
> are doing this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Dufuna-CodeCamp/Dufuna-Fem/pull/580#pullrequestreview-587301333>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARUWXCS4N2IGQLSCKI2IGXLS6IU2ZANCNFSM4XLYZPZA>
> .
>

MoyinoluwaA commented:
It had actually been there for a while and anytime I want to push my work,
I always had to use git add file..and couldn't use git add . Cos of the
file. And I didn't change anything in the person work.

On Wed, 10 Feb 2021, 09:13 Moyinoluwa Adenuga, <dayoh14@gmail.com> wrote:

> Good morning. There was a folder it said I added stuffs and I didn't. So I
> saw your comment on the slack channel. That's why I deleted it.
>
> On Wed, 10 Feb 2021, 07:56 Ireti203, <notifications@github.com> wrote:
>
>> *@Ireti203* commented on this pull request.
>>
>> You deleted some files in another scholar's folder. Is there a reason you
>> are doing this?
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Dufuna-CodeCamp/Dufuna-Fem/pull/580#pullrequestreview-587301333>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ARUWXCS4N2IGQLSCKI2IGXLS6IU2ZANCNFSM4XLYZPZA>
>> .
>>
>
